ktor-network/ktor-network-tls/build.gradle.kts

```diff
@@ -8,6 +10,7 @@ kotlin.sourceSets {
     val jvmTest by getting {
         dependencies {
             api(project(":ktor-network:ktor-network-tls:ktor-network-tls-certificates"))
+            api("io.netty:netty-handler:$netty_version")
```

Comment:
Could we implement the comparison instead of bringing additional dependency here? We want to avoid using additional dependencies here.